assets/airports/eddf.json

```diff
@@ -450,11 +528,11 @@
       "entryPoints": {
         "T173": ["GAPLA", ["DEMAB", "A160-"], ["SOLVU", "A130-"]]
```

Comment:
Created #730 as there are some other issues with this, but the above should resolve the problems here. And in the meantime, `rr GAPLA.KERAX25N.EDDF` is the best bet for getting it to work properly. The `star` command should be fixed soon though.